sprint2.css

```
@@ -0,0 +1,258 @@
+* {
+    box-sizing: 
+    border-box;
```

❗️ 수정요청
가독성을 위해 css 속성과 값은 한줄로 써주세요
```suggestion
    box-sizing: border-box;
```